### DISCUSSION TITLE
Modify Scene Extents

### DISCUSSION THREAD

**Discussion by TJPovey** [assimp/assimp#4084] [Q&A] [answered]
Is it possible to edit the extents of the import prior to creating a scene? 

Typically I have models (fbx/obj) which are far from their origin (0,0,0) - since they are geolocated. Meaning their units are in meters, and the centre of geometry could be around (32000, 32000) or further. 

This causes issues like distortion / faceting of artefacts once exporting the scene. Because of this, I'm hoping that if the extents of the scene can be modified before the scene is created, this issue could be avoided.

Is something like this possible?

**Comment by kimkulling** [comment, marked as answer]
Sure, just go for it. The data is available and you can modify it.

**Reply by TJPovey** [reply]
Can the scene extents be modified using assimp? If so, how? Can you point me to any documentation? I wont be able to modify the extents manually. Thanks
